.github/workflows/agw-workflow.yml

```diff
@@ -0,0 +1,362 @@
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+  pull_request:
+    branches:
+      - master
+      - 'v1.*'
+jobs:
+  lte-test:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      CODEGEN_ROOT: "${{ github.workspace }}/.codegen"
+      PYTHON_BUILD: "${{ github.workspace }}/build"
+      PYTHONPATH: "${{ github.workspace }}"
+      PIP_CACHE_HOME: "${{ github.workspace }}/.pipcache"
+      SWAGGER_CODEGEN_JAR: "${{ github.workspace }}/.codegen/swagger-codegen-cli.jar"
+      MAGMA_DEV_MODE: 1
+      SKIP_SUDO_TESTS: 1
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.5'
+      - name: Install python3-aioeventlet from the magma apt repo
+        run: |
+            cd nms/app
+            # Install python3-aioeventlet from the magma apt repo
+            cat ${{ env.MAGMA_ROOT }}/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub | sudo -E apt-key add -
+            echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" | sudo -E tee /etc/apt/sources.list.d/fbc.list
+            sudo apt-get update -y
+            sudo apt-get install -y python3-aioeventlet
+            sudo rm -rf /var/lib/apt/lists/*
+      - name: Install libraries and dependecies
+        run: |
+            mkdir -p /var/tmp/test_results
+            mkdir -p /var/tmp/codecovs
+            sudo -E apt-get update -y
+            sudo -E apt-get install -y libsystemd-dev pkg-config curl zip unzip
+            sudo -E apt-get install -y virtualenv python-babel python-dev build-essential python3-setuptools python-setuptools autogen autoconf libtool python3-apt python3-requests python3-pip python-protobuf python3-testresources
+      - name: Setup protoc3
+        run: |
+            pip3 install protobuf
+            pip3 install setuptools==49.6.0
+            curl -Lfs https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip -o protoc3.zip
+            unzip protoc3.zip -d protoc3
+            sudo -E mv protoc3/bin/protoc /bin/protoc
+            sudo -E chmod a+rx /bin/protoc
+            # Workaround: the include files need to be found
+            mv ./protoc3/include/google .
+            sudo -E rm -rf protoc3.zip protoc3
+      - name: Setup Swagger
+        run: |
+            mkdir ${{ env.CODEGEN_ROOT }}
+            wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O ${{ env.SWAGGER_CODEGEN_JAR }}
+      - name: Execute test_all
+        run: |
+            make -C ${{ env.MAGMA_ROOT }}/lte/gateway/python test_all
+      - name: Upload Test Results
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: Unit Test Results
+          path: /var/tmp/test_results
+      - uses: codecov/codecov-action@v1
+        with:
+          files: '/var/tmp/codecovs/cover_lte.xml'
```

Comment:
You can specify multiple files if they are comma separated.

Reply:
you mean to combine it with cover_orc8r.xml? sure.. let me do that!